packages/create-bison-app/template/package.json.ejs

```diff
@@ -53,7 +53,9 @@
     "apollo-server-express": "^2.18.1",
 <% } -%>
     "bcryptjs": "^2.4.3",
-    "cross-fetch": "^3.0.5",
+    "cross-fetch": "3.0.5",
```

Comment:
Latest version of cross-fetch creates an issue in the apollo client configuration with the headers.